src/web3/wallet.ts

```diff
@@ -0,0 +1,53 @@
+import Torus from "@toruslabs/torus-embed";
+import Web3 from "web3";
+
+// This function creates the connection to the Torus api.
+//
+// The configuration passed into `torus.init` is what determines which network Torus is on, as well
+// as other block chain information
+export const initializeTorusConnection: () => Promise<Torus> = async (): Promise<Torus> => {
+  const torus = new Torus({
+    buttonPosition: "top-left", // default: bottom-left
+  });
+  await torus.init({
+    buildEnv: "production", // default: production
+    enableLogging: true, // default: false
+    network: {
```

Comment:
so this is already on Polygon's testnet? that's cool

Reply:
I think so(?) I got it directly from the docs